substrate/frame/sudo/src/extension.rs

```diff
@@ -110,6 +111,7 @@ where
 		Ok((
 			ValidTransaction {
 				priority: info.total_weight().ref_time() as TransactionPriority,
+				provides: vec![(who, call).encode()],
```

Comment:
Sometimes tag are prefixed to avoid collision, but here it is not necessary.
```suggestion
				provides: vec![("Sudo", who, call).encode()],
```